demo/state-demo/tls/ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
+MIIDSDCCAjCgAwIBAgIBATANBgkqhkiG9w0BAQsFADBDMRgwFgYDVQQKDA9JbnB1
+dCBPdXRwdXQgSEsxJzAlBgNVBAMMHkNhcmRhbm8gU0wgU2VsZi1TaWduZWQgUm9v
+dCBDQTAeFw0xODA4MDExNjE2MDVaFw0yODA3MjkxNjE3MDVaMEMxGDAWBgNVBAoM
+D0lucHV0IE91dHB1dCBISzEnMCUGA1UEAwweQ2FyZGFubyBTTCBTZWxmLVNpZ25l
+ZCBSb290IENBMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAs566DpR1
+fOkbwXorlfnHvh73C6JEfSh9BMwJg3lGlhiVb6J23/iLp/VlfoYnWm2F6tUR2F+w
+J7khcN+DRWHTd3s3j40QJRaw92CRnKs57s5Yg+rRJaHKytBfBVoit/o5yrMDe/p2
+1B6qWCng8q8N9y9AFqUEuKu/2hRCextf+BgH8vNwJw1VMdqFUWq4nBIqemDP4g0M
+djhuLzbph4w54l4VIoUofqX+7TuN4vNMltOLT46CUZcFxlQrYRqmv3gBK1kVksi7
+OXwhjl062wUHW3jvD3k0bhyF4rd/dvD8JakD5Ef/jKNghicxSJuT6fsNBD1xIREY
+xHPQ/+iRgtMYpQIDAQABo0cwRTAPBgNVHQ8BAf8EBQMDBwYAMBIGA1UdEwEB/wQI
+MAYBAf8CAQAwDQYDVR0OBAYEBGwLqOgwDwYDVR0jBAgwBoAEbAuo6DANBgkqhkiG
+9w0BAQsFAAOCAQEARfX/JWuOQPQyC1fUkFF408Isnwrzv1gy+mbCz5LE+irI6K2K
+cSb1MAU0cqjGhO4I+cQNYybuTZ0sogvhnlhFlS1Q/+UUIvXIJ4KOWKDlkDUi2SJT
+6lR4vplmsdGhDjf2n2CFWlontf58C7A4HarZFzVhXb+yfwSIDEAvxYvKvBtpXOgI
+BIhqLlTBfJvw9Lcg9Lz4bky9nBL7FwHrA4d7f5UiswYVWFNYO2n77KtZoTqAkoiL
+xSC3/BQWPEg0VZ2J+C6c3MUYvhA/ShfgrZJbIOXVACyA7VuphAUWumb28FNUV1+e
+nSGQ9vbeXHp8eKhQoRH46mNQ/8mZmMEPDueOrA==
+-----END CERTIFICATE-----
```

Comment:
Default TLS certificates have been generated for both the `demo` and `wallet-new:test:integration` executable. This way we don't have the extra step of generating them. 
Ideally, I'd like to make `cardano-sl-tools:exe:cardano-x509-certificates` a package on its own which expose a library with a couple of helpers to generate those certificates from the code, without having to dump them to disk. 
That could be part of [(CBR-287]](https://iohk.myjetbrains.com/youtrack/issue/CBR-287).